pkg/controller/logstash/initcontainer.go

```diff
@@ -27,12 +28,12 @@ fi
 
 echo "Setup Logstash configuration"
 
-mount_path=` + InitContainerConfigVolumeMountPath + `
+mount_path=` + volume.InitContainerConfigVolumeMountPath + `
 
 cp -f /usr/share/logstash/config/*.* "$mount_path"
 
-ln -sf ` + InternalConfigVolumeMountPath + `/logstash.yml  $mount_path
-ln -sf ` + InternalPipelineVolumeMountPath + `/pipelines.yml  $mount_path
+ln -sf ` + volume.InternalConfigVolumeMountPath + `/logstash.yml  $mount_path
+ln -sf ` + volume.InternalPipelineVolumeMountPath + `/pipelines.yml  $mount_path
```

Comment:
not from this pr, but should we reuse the constant here?
```suggestion
ln -sf ` + volume.InternalConfigVolumeMountPath + `/` + LogstashConfigFileName + ` $mount_path
ln -sf ` + volume.InternalPipelineVolumeMountPath + `/` + PipelineFileName + ` $mount_path
```